CHANGELOG

```diff
@@ -1,5 +1,7 @@
 0.12.0
 
+  Add support for Python 3.7
```

Comment:
0.12.0 is released and this 3.7 patch is not there. Please remove this paragraph or make a new 0.12.1 block.